src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,177 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> currentNode;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
int value is 0 by default